### PR TITLE
Add test code to cover the branch for find feature

### LINKED ISSUE #198: Lack of test code for find function

### PR DESCRIPTION
**Pull request by wilsonngja**
Fix #198

### CODE REVIEW

File: src/main/java/seedu/duke/parsermanager/ParseFindClient.java [hunk 1]
```diff
@@ -34,14 +35,14 @@ public Command parseCommand() throws DukeException {
         return new CommandFindClient(queryText.trim());
     }
 
-    private void checkCommandValidity(String commandDescription) throws FindIncorrectNumOfTagException,
-            FindEmptyDescriptionException {
+    public void checkCommandValidity(String commandDescription) throws FindIncorrectNumOfTagException,
+            FindEmptyDescriptionException, NoFindClientTagException {
         String[] commandSplitByTag = commandDescription.trim().split(REGEX_CHECKER, 2);
```
Comment (ngdeqi) [inline src/main/java/seedu/duke/parsermanager/ParseFindClient.java:40]: Do avoid magic literals!

File: src/test/java/seedu/duke/FindFunctionTest.java [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+    @Test
+    public void queryName_clientStored_expectCorrectEntries() {
+        ClientList clientList = new ClientList();
+
+        // Store client intended to search
+        storeClient(clientList);
+
+        // This name is not the entire name, but a portion of the name.
+        // It's used to test if not looking for entire name will still produce the correct find result
+        String clientName = "bbles";
```
Comment (ngdeqi) [inline src/test/java/seedu/duke/FindFunctionTest.java:45]: good selection of test case

**Review by ngdeqi — COMMENTED** [review]
LGTM. Only has a very minor code quality violation

**Review by OVReader — APPROVED** [review]
Overall LGTM with the addition of relevant test cases

**Pull request merged**